tests/ecosystem/upgrade/test_upgrade_ocp.py

```diff
@@ -51,6 +51,16 @@ def test_upgrade_ocp(self):
             image_path = config.UPGRADE['ocp_upgrade_path']
             self.cluster_operators = ocp.get_all_cluster_operators()
             logger.info(f" oc version: {ocp.get_current_oc_version()}")
+            # Verify Upgrade subscription channel:
+            for sampler in TimeoutSampler(
+                timeout=250,
+                sleep=120,
```

Comment:
let's use shorter sleep time in case it will be ready before 2 minutes for each sleep - cost us nothing.
I would use 10

Reply:
2 minutes is according to OCP upgrade instructions (for testing)

Reply:
@AviadP it's about polling rate. Does OCP upgrade instructions for testing specify that polling should be done in an interval of 2 minutes?